README.md

```diff
@@ -224,6 +225,7 @@ For complete usage of go-github, see the full [package docs][].
 [package docs]: https://godoc.org/github.com/google/go-github/github
 [GraphQL API v4]: https://developer.github.com/v4/
 [shurcooL/githubv4]: https://github.com/shurcooL/githubv4
+[documentation]: https://pkg.go.dev/github.com/google/go-github/v32/github?tab=doc
```

Comment:
Please delete this line and change the URL above on line 225 to this one:

```
[package docs]: https://pkg.go.dev/github.com/google/go-github/v32/github
```

Let's also get rid of the "?tab=doc" because that is added by the backend and may change.

Reply:
Done :)